requirements.txt

```diff
@@ -1,3 +1,4 @@
 ansible==4.10.0
 ansible-core==2.11.12
-awscli==1.19.48
\ No newline at end of file
+awscli==1.19.48
+jmespath==1.0.1
```

Comment:
Added jmespath to the requirements file as this is required by ansible to decode the fence_data.json file